spec/features/user_signs_in_spec.rb

```diff
@@ -5,25 +5,23 @@
   I want to sign in
   So that I can regain access to my account
 } do
-  # scenario 'specify valid credentials', js: true do
-    # user = FactoryGirl.create(:user)
+  scenario 'specify valid credentials' do
+    user = FactoryGirl.create(:user)
 
-    # visit new_user_session_path
+    visit new_user_session_path
 
-    # fill_in 'Email', with: user.email
-    # fill_in 'Password', with: user.password
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
 
-    # click_button 'Log in'
+    click_button 'Log in'
 
-    # expect(page).to have_content('Signed in successfully')
-    # expect(page).to have_content('Sign Out')
-  # end
+    expect(page.source).to match(/Signed in successfully/)
+  end
 
-  # scenario 'specify invalid credentials', js: true do
-    # visit new_user_session_path
+  scenario 'specify invalid credentials' do
+    visit new_user_session_path
 
-    # click_button 'Log in'
-    # expect(page).to have_content('Invalid email or password')
-    # expect(page).to_not have_content('Sign Out')
-  # end
+    click_button 'Log in'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.